compiler/optimizer/CopyPropagation.cpp

```diff
@@ -745,16 +745,18 @@ int32_t TR_CopyPropagation::perform()
    for (auto itr = usesToBeFixed.begin(), end = usesToBeFixed.end(); itr != end; )
       {
       int32_t fixUseIndex = itr->first;
+      TR::Node *fixUseNode = useDefInfo->getNode(fixUseIndex);
       invalidateDefUseInfo = true;
+      if (useDefInfo->hasLoadsAsDefs())
```

Comment:
This condition short cuts the potentially expensive loop. The loop is only necessary if loads are treated as defs and can therefore be avoided if not operating in that mode.